MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -441,35 +712,58 @@ int main(int argc, char * argv[])
                                     MsixCoreLib::UnmountVHD(unpackDestination);
                                     if (_wremove(unpackDestination.c_str()) != 0)
                                     {
+                                        errorDesc += L"Failed best-effort attempt to delete the incomplete VHD(X) file.";

```

Comment:
Same as above

Reply:
As discussed, all the Telemetry variables will have the same Error Message as the actual Console statements. The change for outputting the Error Message on Console from a variable will be done as part of Error Message Improvement Task that will be taken later.